tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,34,35,py}-django{18,19,110},isort
+envlist = py{310, py}-django{42},isort
```

Comment:
Graag ook Py 3.9 en Django 3.2 hier in opnemen, als library kun je niet uitgaan van 1 en maar 1 versie van Python en Django.

Reply:
Ik heb ook de issue gefixt met de tox tests, door een Python 3.9 omgeving te installeren.